tests/plan/import/main.fmf

```diff
@@ -5,3 +5,7 @@
 /errors:
   summary: Check correct error handling in invalid situations
   test: ./errors.sh
+
+/mod-plan:
```

Comment:
```suggestion
/modify:
```

Let's call the test `/tests/plan/import/modify` to make it a bit more clear at the first sight (`mod` is a bit confusing and `plan` is already included in the name).